fossunited/api/chapter.py

```
@@ -125,9 +127,15 @@ def generate_ics(event_ids):
 
     c = Calendar()
 
+    filters = [["name", "IN", ids], ["status", "=", "Live"], ["is_published", "=", 1]]
+    if chapter:
+        if not isinstance(chapter, str) or len(chapter) > 140:
+            frappe.throw(_("Invalid chapter"), frappe.ValidationError)
```

Ruff (0.15.4)

[error] Undefined name `_`

(F821)